pkg/argocd/applications.go

```diff
@@ -44,8 +35,15 @@ func PullApplication(apiClient *clients.Settings, name, nsname string) (*Applica
 		return nil, fmt.Errorf("application 'apiClient' cannot be empty")
 	}
 
+	err := apiClient.AttachScheme(argocdtypes.AddToScheme)
+	if err != nil {
+		glog.V(100).Info("Failed to add Application scheme to client schemes")
```

Comment:
Maybe include argocd in the message so it is clear which application resource the used is being notified of
```suggestion
		glog.V(100).Info("Failed to add argocd application scheme to client schemes")
```